avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
sure, you don't need to update this PR, feel free to send another one and only share the link here... I'm still a bit worried it's not the final fix, but hopefully it is. At least it should improve the situation. I'd prefer if we found a reproducer, but I tried my best and couldn't do it.

As for the early status, I had the tracebacks where the main thread was interrupted after the early_status, therefor it had to retrieve it.